test_scripts/Defects/4_5/1921_Invalid_PT_after_cutting_unknow_values.lua

```diff
@@ -125,18 +147,16 @@ local function contains(pTbl, pValue)
   return false
 end
 
---[[ @CheckCuttingUnknowValues: Perform app registration, PTU and check absence of unknown values in
---! OnPermissionsChange notification
+--[[ @CheckCuttingUnknowValues: expectation of OnPermissionsChange notification and check its content
 --! @parameters:
+--! pNotifTimes - expected number of OnPermissionsChange notification
 --! self - test object
 --! @return: none
 --]]
-local function CheckCuttingUnknowValues(ptuUpdateFunc, self)
-  commonDefects.rai_ptu_n_without_OnPermissionsChange(1, ptuUpdateFunc, self)
+local function CheckCuttingUnknowValues(pNotifTimes, self)
```

Comment:
```suggestion
local function CheckCuttingUnknownValues(pNotifTimes, self)
```

Reply:
@jacobkeeler Please find update in d45c6c1